### DISCUSSION TITLE
Nuxt, Unocss and histoire

### DISCUSSION THREAD

**Discussion by husayt** [unocss/unocss#1690] [Q&A]
Has anyone tried using histoire and nuxt with unocss. Any tips would be appreciated. I have tried multiple approaches, But histoire is not picking unocss styles. Even if you managed to get it working without Nuxt just unocss with histoire, that will also be interesting. Thanks

**Comment by piscis** [comment]
Sorry to resurrect this discussion. But after searching for ours I also came about this thread a couple of times. A possible workaround to get things started can be found here: https://github.com/histoire-dev/histoire/discussions/113#discussioncomment-6657134
